hydrolib/core/io/rr/models.py

```diff
@@ -1,220 +1,333 @@
-    ow_levels: Optional[Path] = Path("ow_level.his")
-    balance_file: Optional[Path] = Path("3b_bal.out")
-    his_3B_area_length: Optional[Path] = Path("3bareas.his")
-    his_3B_structure_data: Optional[Path] = Path("3bstrdim.his")
-    his_rr_runoff: Optional[Path] = Path("rrrunoff.his")
-    his_sacramento: Optional[Path] = Path("sacrmnto.his")
-    his_rwzi: Optional[Path] = Path("wwtpdt.his")
-    his_industry: Optional[Path] = Path("industdt.his")
-    ctrl_ini: Optional[Path] = Path("ctrl.ini")
-    root_capsim_input_file: Optional[Path] = Path("root_sim.inp")
-    unsa_capsim_input_file: Optional[Path] = Path("unsa_sim.inp")
-    capsim_message_file: Optional[Path] = Path("capsim.msg")
-    capsim_debug_file: Optional[Path] = Path("capsim.dbg")
-    restart_1_hour: Optional[Path] = Path("restart1.out")
-    restart_12_hours: Optional[Path] = Path("restart12.out")
-    rr_ready: Optional[Path] = Path("RR-ready")
-    nwrw_areas: Optional[Path] = Path("NwrwArea.His")
-    link_flows: Optional[Path] = Path("3blinks.his")
-    modflow_rr: Optional[Path] = Path("modflow_rr.His")
-    rr_modflow: Optional[Path] = Path("rr_modflow.His")
-    rr_wlm_balance: Optional[Path] = Path("rr_wlmbal.His")
-    sacramento_ascii_output: Optional[Path] = Path("sacrmnto.out")
-    nwrw_input_dwa_table: Optional[Path] = Path("pluvius.tbl")
-    rr_balance: Optional[Path] = Path("rrbalans.his")
-    green_house_classes: Optional[Path] = Path("KasKlasData.dat")
-    green_house_init: Optional[Path] = Path("KasInitData.dat")
-    green_house_usage: Optional[Path] = Path("KasGebrData.dat")
-    crop_factor: Optional[Path] = Path("CropData.dat")
-    crop_ow: Optional[Path] = Path("CropOWData.dat")
-    soil_data: Optional[Path] = Path("SoilData.dat")
-    dio_config_ini_file: Optional[Path] = Path("dioconfig.ini")
-    bui_file_for_continuous_calculation_series: Optional[Path] = Path("NWRWCONT.#")
-    nwrw_output: Optional[Path] = Path("NwrwSys.His")
-    rr_routing_link_definitions: Optional[Path] = Path("3b_rout.3b")
-    cell_input_file: Optional[Path] = Path("3b_cel.3b")
-    cell_output_file: Optional[Path] = Path("3b_cel.his")
-    rr_simulate_log_file: Optional[Path] = Path("sobek3b_progress.txt")
-    rtc_coupling_wq_salt: Optional[Path] = Path("wqrtc.his")
-    rr_boundary_conditions_sobek3: Optional[Path] = Path("BoundaryConditions.bc")
-
-    rr_ascii_restart_openda: Optional[Path] = None
-    lgsi_cachefile: Optional[Path] = None
-    meteo_input_file_rainfall: Optional[Path] = None
-    meteo_input_file_evaporation: Optional[Path] = None
-    meteo_input_file_temperature: Optional[Path] = None
+    _validate_mappix_paved_area_sewage_storage = _validator_mappix_value(
+        "mappix_paved_area_sewage_storage", _mappix_paved_area_sewage_storage_name
+    )
+    mappix_paved_area_sewage_storage: ImmutableDiskOnlyFileModel = Field(
+        ImmutableDiskOnlyFileModel(Path(_mappix_paved_area_sewage_storage_name)),
+        allow_mutation=False,
```

Comment:
this is the arguments default value already, but I guess you intentionally want to make that explicit here?

Reply:
I think it is better to omit it, it kinda raises the impression that `allow_mutation=True` is a valid usage of the class, but the class we are using is `ImmutableDiskOnlyFileModel` => it should be implied that `allow_mutation=False`.

Reply:
> this is the arguments default value already, but I guess you intentionally want to make that explicit here?

Note that this `allow_mutation` is set on the `Field` function, where the default value is `True`. By setting it to `False` we provide a faux immutability to the field, meaning that users cannot assign a new (different) model to the `Field`.
This is different from the `allow_mutation` set on the model, which specifies that all fields of the `ImmutableDiskOnlyFileModel` are faux immutable. By specifying both users cannot adjust the model fields nor the model itself on the `RainfallRunoffModel`. 

As such it is explicitly required to have it here.